test/unit/models/hospital.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
```

Comment:
Here's a first test that verifies that we can properly create a Hospital record.